client/src/main/scala/com/lunatech/undoredo/recipes/views/RecipesView.scala

```diff
@@ -0,0 +1,113 @@
+package com.lunatech.undoredo.recipes.views
+
+import japgolly.scalajs.react._
+import japgolly.scalajs.react.vdom.html_<^._
+
+import com.lunatech.undoredo.recipes.Client
+import com.lunatech.undoredo.recipes.models._
+import com.lunatech.undoredo.recipes.handlers.RecipesCircuit
+
+object RecipesView {
+
+  private val couscous = Recipe(
+    "couscous",
+    "Couscous",
+    Seq("ig12", "ig22"),
+    Seq("is12", "is22")
+  )
+  private val cremeBrulee = Recipe(
+    "creme-brulee",
+    "Creme Brulee",
+    Seq("ig13", "ig23"),
+    Seq("is13", "is23")
+  )
+  case class Props(
+      model: RootModel
+  )
+
+  private val component = ScalaComponent
+    .builder[Props]("recipesView")
+    .render_P { props: Props =>
+      <.div(
+        <.div(
+          ^.cls := "btn-group",
+          <.button(
+            ^.`type` := "button",
+            ^.cls := "btn btn-outline-secondary",
+            ^.onClick --> Callback(RecipesCircuit.dispatch(UndoRedoHistoryActions.Undo)),
```

Comment:
would be nice to have a dipatchCb in RecipesCircuit to avoid creating a Callback at every dispatch